src/pages/library/Showcase.js

```diff
@@ -16,104 +18,337 @@ import * as Style from 'src/libs/style'
+
+      // Trim items from the sidebar for which there aren't any featured workspaces.
+      const activeTags = _.keys(workspacesByTag)
+      const activeSections = _.flow([
+        _.map(_.update(['labels'], labels => _.intersectionBy(_.toLower, labels, activeTags))),
+        _.map(section => ({ ...section, tags: _.map(_.toLower, section.labels) })),
+        _.remove(section => _.isEmpty(section.labels))
+      ])(sidebarSections)
+
+      setFeaturedList(featuredWorkspaces)
+      setWorkspacesByTag(workspacesByTag)
+      setSections(activeSections)
+
+      // TODO: is this actually doing anything for us?
+      StateHistory.update({ featuredWorkspaces })
     }
 
     loadData()
   })
 
-  const allFeatured = _.intersectionWith(
-    ({ workspace }, featured) => workspace.namespace === featured.namespace && workspace.name === featured.name,
-    workspaces,
-    featuredList)
-
-  const { bestPractices, newAndInteresting, featured } = _.groupBy(
-    ({ workspace: { namespace, name, attributes: { description } } }) => Utils.cond(
-      [description?.startsWith('### GATK Best Practices'), () => 'bestPractices'],
-      [_.find({ namespace, name }, featuredList)?.isNew, () => 'newAndInteresting'],
-      () => 'featured'
-    ),
-    allFeatured
-  )
+  const filterBySections = workspaces => {
+    if (_.isEmpty(selectedSections)) {
+      return workspaces
+    } else {
+      const tags = _.uniq(_.flatMap(section => section.tags, selectedSections))
+      return _.uniq(_.flatMap(tag => workspacesByTag[tag], tags))
+    }
+  }
+  // eslint-disable-next-line lodash-fp/no-single-composition
+  const filterByTags = workspaces => _.flow(_.map(tag => _.intersection(workspacesByTag[tag]), selectedTags))(workspaces)
+  const filterByText = workspaces => {
+    const lowerSearch = _.toLower(searchFilter)
+    return _.isEmpty(lowerSearch) ?
+      workspaces :
+      _.filter(workspace => _.includes(lowerSearch, workspace.lowerName) || _.includes(lowerSearch, workspace.lowerDescription), workspaces)
+  }
+  const filteredWorkspaces = _.flow([
+    filterBySections,
+    filterByTags,
+    filterByText
+  ])(featuredList)
 
   return h(FooterWrapper, { alwaysShow: true }, [
-    libraryTopMatter('showcase & tutorials'),
-    div({ role: 'main', style: { margin: '2.5rem 2rem', flexGrow: 1 } }, [
-      !(featuredList && workspaces) ?
-        centeredSpinner() :
-        div({ style: { display: 'flex' } }, [
-          !_.isEmpty(newAndInteresting) && div({ style: styles.column }, [
-            div({ style: styles.header }, 'New and interesting'),
-            ..._.map(makeCard('new'), newAndInteresting)
+    libraryTopMatter('featured workspaces'),
+    !featuredList ?
+      centeredSpinner() :
+      h(Fragment, [
+        div({ style: { display: 'flex', margin: '1rem 1rem 0', alignItems: 'baseline' } }, [
+          div({ style: { width: '19rem', flex: '0 0 auto' } }, [
```

Comment:
Since 'none' is a synonym for '0 0 auto', and the former is used above, I wonder if it makes sense to always use one or the other?